tui_main.c

```diff
@@ -1278,39 +1278,39 @@ typedef enum {
   Ged_dir_right,
 } Ged_dir;
 
-void ged_dir_input(Ged* a, Ged_dir dir) {
+void ged_dir_input(Ged* a, Ged_dir dir, int step_length) {
```

Comment:
can you use `Isz` instead of int ? Orca-c uses its own signed and unsiged integer types for better portability across platforms